modules/end-user-guide/pages/developer-workspaces.adoc

```diff
@@ -14,6 +14,21 @@
 * Tool dependencies needed by developers to work on a project
 * Application runtime: a replica of the environment where the application runs in production
 
-Pods manage each component of a {prod-short} workspace. Therefore, everything running in a {prod-short} workspace is running inside containers. This makes a {prod-short} workspace highly portable.
+Pods manage each component of a {prod-short} workspace.
+Therefore, everything running in a {prod-short} workspace is running inside containers.
+This makes a {prod-short} workspace highly portable.
 
-The embedded browser-based IDE is the point of access for everything running in a {prod-short} workspace. This makes a {prod-short} workspace easy to share.
+The embedded browser-based IDE is the point of access for everything running in a {prod-short} workspace.
+This makes a {prod-short} workspace easy to share.
+
+Visual Studio Code is the default browser-based IDE. {prod-short} adds these features:
+
+Tasks::
+Commands that are defined in the devfile appear as tasks.
++
+To execute such a command, go to menu:Terminal[Run Task].
++
+Alternatively, kbd:[F1] to open the *Command Palette* and type `Tasks: Run Tasks` to see a drop-down list of available task-related commands.
+
+Process monitoring::
```

Comment:
There are 2 distinct VS Code built-in extensions: one that "monitors resources (CPU and RAM)", and another one "exposes ports". You are talking about the second one here but it can be confused with the resource monitoring.